.github/workflows/main.yml

```diff
@@ -1,16 +1,18 @@
 name: Java CI
 
-on: [push]
+on: [push, pull_request]
```

Comment:
@borisf I added `pull_request` so that pull requests will also have CI feedback.

Reply:
Thanks